src/BenchmarkDotNet/Templates/WasmCsProj.txt

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="$SDKNAME$" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>$TFM$</TargetFramework>
+    <TargetArchitecture>wasm</TargetArchitecture>
+    <TargetOS>Browser</TargetOS>
+    <RuntimePackDir>$RUNTIMEPACKDIR$</RuntimePackDir>
```

Comment:
this doesn't need to be set since doing a `dotnet publish` on the csproj will automatically pull down the runtime pack and put all assets into the publish directory and we can take everything from there.